redfish/provider/resource_redfish_boot_order_test.go

```diff
@@ -38,6 +38,23 @@ func TestAccRedfishBootOrderOptions_basic(t *testing.T) {
 	})
 }
 
+// Test to import Boot Order - positive
+func TestAccRedfishBootOrder_Import(t *testing.T) {
```

Comment:
Instead of creating a new import function, can we interweave the import checks between the existing tests?
Eg. https://github.com/dell/terraform-provider-redfish/blob/main/redfish/provider/resource_redfish_virtual_media_test.go#L248
This ensures real-world config to state matching.

Reply:
For instance, from a preliminary look at the code, I think it may not match the state if the check is put here: https://github.com/dell/terraform-provider-redfish/pull/161/files#diff-301772d2a0e7085770f78ea6bcd7958ddbd018946f549878528aa60c85835f77R20